dotnet/orleans
# Minor documentation fixes

Most of the fixes were converting `<T>` to `&lt;T&gt;` so the generic type references would be rendered as intended.

Alternatively, we could backtick quote all inline references such as type names, interface names, method names, keywords, etc. and then they don't need to be escaped if they contain angle brackets.  Let me know if you would like me to do that, instead.

Manual testing
--------------

1. Made my edits using MardownPad 2, which has a real-time preview pane, allowing me to verify my fixes.
2. Ran `jekyll build` and opened a few of the affected pages under `_site/` to confirm my fixes were applied the same way as in MarkdownPad 2.

Mission accomplished!
